src/browser/fetch/form_data.rs

```diff
@@ -60,9 +61,12 @@ impl FormData {
     #[allow(clippy::missing_panics_doc)]
     pub fn append_json<T>(&mut self, name: &str, data: &T) -> Result<()>
     where
-        T: Serialize + ?Sized,
+        T: Serialize,
     {
-        let str = serde_json::to_string(data).map_err(FetchError::SerdeError)?;
+        // FIXME Can a different `append` be used to append a `JsValue` directly?
```

Comment:
Could you rewrite all `FIXME`s to `@TODO` for consistency so we can search through all todos/fixmes with one term?